allensdk/brain_observatory/behavior/behavior_project_cache/behavior_neuropixels_project_cache.py

```diff
@@ -0,0 +1,63 @@
+from allensdk.brain_observatory.behavior.behavior_project_cache.\
+    project_apis.data_io import VisualBehaviorNeuropixelsProjectCloudApi
+from allensdk.brain_observatory.behavior.behavior_project_cache.\
+    project_cache_base import ProjectCacheBase
+
+
+class VisualBehaviorNeuropixelsProjectCache(ProjectCacheBase):
+
+    PROJECT_NAME = "visual-behavior-ecephys-data"
+    BUCKET_NAME = "visual-behavior-ecephys"
+
+    def __init__(
+            self,
+            fetch_api: VisualBehaviorNeuropixelsProjectCloudApi,
+            fetch_tries: int = 2,
+            ):
+
+        super().__init__(fetch_api=fetch_api, fetch_tries=fetch_tries)
+        """ Entrypoint for accessing Visual Behavior Neuropixels data.
```

Comment:
This docstring should be above the invocation of `super()` (unless the point is to have `help()` find the docstring for `ProjectCacheBase`).

Reply:
moved